server/src/controllers/authentication.js

```diff
@@ -3,12 +3,18 @@ import User from '../models/user';
 
 export default {
     signup : (req, res, next) => {
-        const { email, password, firstName, lastName } = req.body;
+        const {
```

Comment:
Set up the sign up page also with the required fields. Add those commits to this pull.